bin/release.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+set -x
+
+function usage() {
+  echo "$0 \
+    -i <gcloud project ID name> \
+    -t <tag name to apply to artifacts>"
+  exit 1
+}
+
+# Initialize variables
+PROJECT_ID=""
+TAG_NAME=""
+
+# Handle command line args
+while getopts i:t: arg ; do
+  case "${arg}" in
+    i) PROJECT_ID="${OPTARG}";;
```

Comment:
I don't think that passing the PROJECT_ID makes sense.

For docker we need to push to gcr.io/istio-io and docker.io/istio, for istioctl let's use istio-release bucket.